library/navigation/src/main/java/com/epmedu/animeal/navigation/route/TabsRoute.kt

```diff
@@ -0,0 +1,10 @@
+package com.epmedu.animeal.navigation.route
```

Comment:
Good, but we forgot to move SignUpRoute to common module. Can you move TabsRoute there please?